Composer/packages/server/src/models/bot/builder.ts

```diff
@@ -98,6 +102,59 @@ export class Builder {
     this._locale = v;
   }
 
+  /**
+   * Orchestrator: Get available list of NLR models
+   */
+  public async runOrchestratorNlrList(): Promise<IOrchestratorNLRList> {
+    return JSON.parse(await Orchestrator.nlrListAsync());
+  }
+
+  /**
+   * Orchestrator: Download an available NLR model.
+   *
+   * @remarks Available NLR models and VersionIds are obtained by running runOrchestratorNlrList first.
+   *
+   * @param modelPath - Folder path to save NLR model
+   * @param nlrId - VersionId of the model
+   * @param onProgress - Callback to notify of D/L progress
+   * @param onFinish - Callback to notify of D/L completed
+   */
+  public async runOrchestratorNlrGet(
+    modelPath: string,
+    nlrId: string,
+    onProgress: IOrchestratorProgress,
+    onFinish: IOrchestratorProgress
+  ): Promise<void> {
+    await Orchestrator.nlrGetAsync(modelPath, nlrId, onProgress, onFinish);
+  }
+
+  /**
+   * Orchestrator: Build command to compile .lu files into Binary LU (.blu) snapshots.
```

Comment:
>.lu [](start = 44, length = 3)

Tien mentioned that qna files are also supported built into blu snapshots, so I think .qna files should also be included.

Reply:
I'm not sure about qna support in orchestrator. Have you tried that @feich-ms?

Reply:
Get it, saw the samples, the qna support just means it can take the questions as intent "QnaMaker", it's not the way we expected to integrate with Qna service or AdaptiveQnaRecoginzer. So for now, composer should only deal with .lu.